src/components/screens/Connect.tsx

```diff
@@ -53,67 +64,7 @@ export function Connect({ mode, initialURI }: Props) {
     navigator.userAgent?.includes("; wv") &&
```

Comment:
we could be use `useMemo` here

Reply:
correct or we can just put that outside the component actually